### PR TITLE
docs: expansions and CasedKeys store

### PR DESCRIPTION
**Pull request by mcdurdin**
Goes with keymanapp/keyman#4584 and keymanapp/keyman#4586.

### CODE REVIEW

File: developer/language/guide/history.md [hunk 1]
```diff
@@ -81,3 +81,8 @@ title: Keyman language history
 ## Keyman 13
 
 *   Virtual keys no longer need to be in quotes for the [`&hotkey`](../reference/hotkey) store
+
+## Keyman 14
+
+*   Added [range expansions](expansions)
+*   Added [&CasedKeys store](../referencecasedkeys)
```
Comment (darcywong00) [inline developer/language/guide/history.md:88]: Do you need to attach these 2 new files?

Comment (mcdurdin) [reply developer/language/guide/history.md]: Sometimes `git commit -am` bites me

**Review by darcywong00 — APPROVED** [review]
lgtm

**Review by jahorton — CHANGES_REQUESTED** [review]
Please address this comment; I don't see any reference of ISO name caveats.

https://github.com/keymanapp/keyman/pull/4584#issuecomment-790264180

**Comment by mcdurdin** [comment]
> Please address this comment; I don't see any reference of ISO name caveats.

"ranges are based on the numeric value of the virtual key as defined by Windows, not its position on the keyboard"

But I can make it more explicit.

**Review by jahorton — APPROVED** [review]

**Pull request merged**